scripts/deployment/deploy_loanToken.py

```diff
@@ -93,57 +93,55 @@ def deployLoanToken(acct, sovryn, loanTokenAddress, loanTokenSymbol, loanTokenNa
         [loanTokenAddress]
     )
 
-    print("Setting up margin pool params on loan token.")
+    if not collateralAddresses:
+        collateralAddresses = []
 
     constants = shared.Constants()
-    params = [];
-
-    data = [
-        b"0x0", ## id
-        False, ## active
-        str(acct), ## owner
-        constants.ZERO_ADDRESS, ## loanToken -> will be overwritten
-        collateralAddress, ## collateralToken.
-        Wei("20 ether"), ## minInitialMargin -> 20% (allows up to 5x leverage)
-        Wei("15 ether"), ## maintenanceMargin -> 15%, below liquidation
-        0 ## fixedLoanTerm -> will be overwritten with 28 days
-    ]
-
 
-    params.append(data)
+    print("Setting up margin pool params on loan token.")
 
-    #configure the token settings
-    calldata = loanTokenSettings.setupLoanParams.encode_input(params, False)
+    params = []
     
-    #set the setting contract address at the loan token logic contract (need to load the logic ABI in line 171 to work)
-    tx = loanToken.updateSettings(loanTokenSettings.address, calldata, { "from": acct })
+    for collateralAddress in collateralAddresses:
+        data = [
+            b"0x0", ## id
+            False, ## active
+            str(acct), ## owner
+            constants.ZERO_ADDRESS, ## loanToken -> will be overwritten
+            collateralAddress, ## collateralToken.
+            Wei("20 ether"), ## minInitialMargin -> 20% (allows up to 5x leverage)
+            Wei("15 ether"), ## maintenanceMargin -> 15%, below liquidation
+            0 ## fixedLoanTerm -> will be overwritten with 28 days
+        ]
+
+        params.append(data)
+
+    #configure the token settings, and set the setting contract address at the loan token logic contract
+    tx = loanToken.setupLoanParams(params, False)
```

Comment:
One question, Wukong. Here, you pass (params, False). Before it was 
```calldata = loanTokenSettings.setupLoanParams.encode_input(params, True)```
What difference does it make?

Reply:
before:
`calldata = loanTokenSettings.setupLoanParams.encode_input(params, False)`
`tx = loanToken.updateSettings(loanTokenSettings.address, calldata, { "from": acct })`

now:
`tx = loanToken.setupLoanParams(params, False)`

nothing different. just directly call setupLoanParams() of loanToken

Reply:
True != False.

Reply:
look at the params

Reply:
sorry, my bad. i was comparing the wrong lines. everything is fine!